config/locales/project.en.yml

```diff
@@ -88,6 +88,11 @@ en:
         title: School details
         rows:
           original_school_name: Original school name
+          school_name_link_to_gias:
+            html: |
+              %{school_name}<br/>
+              %{link_to_gias}
+          view_in_gias: View this school's information in GIAS (opens in a new tab)
```

Comment:
```suggestion
          view_in_gias: View the school's information in GIAS (opens in new tab)
```

Reply:
I have suggested a little tweak… we tend to refer to `the school's` rather than `this school's` (add project page).
Also external links should be appended with `(opens in new tab)`

Also is it possible to add a line break after the school name to force the GIAS link to the line below even if the school has a very short name? I think this would make the School name more readable than having it all on the one line.

Reply:
There's a break in the locale string on line 93 of `project.en.yml`, which will do this